src/lib/Components/Bidding/Components/Input.tsx

```diff
@@ -4,6 +4,9 @@ import { TextInput, TextInputProps } from "../Elements/TextInput"
 
 interface InputProps extends TextInputProps, TextInputProperties {
   error?: boolean
+  refName?: any
+  createCustomRef?: any
```

Comment:
IMO, this code should probably have comments because it's pretty far from the normal react flow

Reply:
added :)